File: Number Pattern/Number Pattern.py
arr = []
N=6
for i in range(1,N+1):
    p=''
    for j in range(1,i+1):
        p+=str(j)
    for k in range (i-1,0,-1):
        p+=str(k)
    arr.append(int(p))
print(arr)

File: Number Pattern/Number Pattern.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main(){

vector<string> arr;
for(int i=1;i<N+1;i++){
    string p="";
    for(int j=1;j<i+1;j++){
        p+=to_string(j);
    }
    for(int l=i-1;l>0;l--){
        p+=to_string(l);
    }
    arr.push_back((p));
    
    
}
cout<<arr;
}